### ISSUE TITLE
Can not open the source file "fl/Headers.h"

### ISSUE THREAD

**Issue by HongyuanChen** [fuzzylite/fuzzylite#124]
Dear author :
 I have download the Sources and Binaries – Version 6.0 in unix. I have copy the bin folder with libfuzzylite.so in it.
 I have add the lib path and name in the cmakelist.txt like:include_directories(/.../bin) link_libraries(libfuzzylite). In the main.cpp
when #include "fl/Headers.h". the Vscode has reported such error : Can not open the source file "fl/Headers.h". Have I misunderstood the use of this binary lib?
Thank you for your attention, dear author.

**Comment by HongyuanChen** [comment]
Sorry to bother you. I have solved the question above. But I have faced a new one.
When I want to present a rule like “if A is a and B is b then C is c”,just copying the code listed above in c++, it may show the error :Rule block 1 <mamdani> has no conjunction operator.
When I look into the rule.h, it list as follows: 
`if variable is [hedge]* term [(and|or) variable is [hedge]* term]*`
`then variable is [hedge]* term [and variable is [hedge]* term]* [with w]?`
where elements in brackets are optional, elements in parentheses are compulsory, `*`-marked elements may appear zero or more times, and `?`-marked elements may appear once or not at all.
I can't see it different from what I have typed:
mamdani->addRule(Rule::parse("if obstacle is left and obstacle1 is left then mSteer is right", engine));
May someone willing to help me sove this problem?
